apps/notifications/src/alert/services/deployment/deployment.service.ts

```diff
@@ -0,0 +1,67 @@
+import { Injectable } from '@nestjs/common';
+import { ConfigService } from '@nestjs/config';
+import axios from 'axios';
+
+import { LoggerService } from '@src/common/services/logger.service';
+import { GlobalEnvConfig } from '@src/config/env.config';
+
+@Injectable()
+export class DeploymentService {
+  constructor(
+    private readonly configService: ConfigService<GlobalEnvConfig>,
+    private readonly loggerService: LoggerService,
+  ) {
+    this.loggerService.setContext(DeploymentService.name);
+  }
+
+  async getDeploymentBalance(
+    owner: string,
+    dseq: string,
+  ): Promise<{ balance: number } | null> {
+    const { data: result } = await axios.get<any>(
+      `${this.configService.getOrThrow('API_NODE_ENDPOINT')}/akash/deployment/v1beta3/deployments/info`,
```

Comment:
question (non-blocking): should we put this in the http sdk package?

Reply:
we should and there is such already. but there is an issue with modules resolution. I can't import non commonjs files into this project and I don't want to get distracted into this atm